databases/week3/WEEK3 homework/mealSharing_db

```diff
@@ -0,0 +1,256 @@
+CREATE DATABASE Meal_Sharing;
+
+
+CREATE TABLE `Meal` (
+  `id` int UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(255) NOT NULL,
+  `description` text,
+  `location` varchar(255) NOT NULL,
+  `Hosting_time` datetime NOT NULL,
+  `max_reservations` int NOT NULL,
+  `price` decimal NOT NULL,
```

Comment:
Don't forget to specify how many digits and how many decimal places can the number have.

Reply:
thank you for comments, specially `.SQL` tip , I would really appreciate it , if you can tell me what do you mean by formating?

Reply:
I mean stuff like this, for instance, is hard to read:
![image](https://user-images.githubusercontent.com/609864/155312245-4f25c90d-9355-47ca-90f4-f910d168ec28.png)

Reply:
This is easier to read (nevermind having each value in a different lines, the consistency is the important part):
![image](https://user-images.githubusercontent.com/609864/155312402-8e9a577f-1904-4597-ab57-52f9b0ee057f.png)

Reply:
Does it make more sense now?